src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -37,4 +37,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageReference Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and !$(RuntimeFrameworkVersion.StartsWith('1.'))">
+    <PackageReference Include="Microsoft.NETCore.DotNetAppHost" Version="$(RuntimeFrameworkVersion)" IsImplicitlyDefined="true" />
```

Comment:
For the reason I explained above - it is not about external facing scenario but to ensure that we get the appropriate host restore for sharedFX build. I concur it can be done differently for 2.0 but that does not affect the change here in SDK. Even so, I believe I am agreeing with your original comment that, given dotnet.exe/apphost.exe are referenced in m.n.app already, @ramarag may not need this change here.